src/libraries/System.Private.Xml/src/System/Xml/Dom/DocumentXPathNavigator.cs

```diff
@@ -180,6 +182,8 @@ public override string Value
                     case XmlNodeType.SignificantWhitespace:
                         return ValueText;
                     default:
+                        Debug.Assert(_source.Value != null);
+                        // TODO: It could be better to switch this to nullable even if that implies switching it in the base type.
```

Comment:
```suggestion
                        // TODO-NULLABLE: It could be better to switch this to nullable even if that implies switching it in the base type.
```